easybuild/easyconfigs/c/canu/canu-2.1.1-GCCcore-9.3.0-Java-13.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'ConfigureMake'
+
+name = 'canu'
+version = '2.1.1'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://canu.readthedocs.io'
+description = "Canu is a fork of the Celera Assembler designed for high-noise single-molecule sequencing"
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+github_account = 'marbl'
+source_urls = ['https://github.com/marbl/canu/releases/download/v%(version)s']
+sources = [SOURCE_TAR_XZ]
+checksums = ['18c030ada93286be90c364387879025c17001c8e445e64719c866bc6c7609b98']
+
+builddependencies = [
+    ('binutils', '2.34'),
+    ('git', '2.23.0', '-nodocs'),
+]
+
+dependencies = [
+    ('Java', '13', '', True),
```

Comment:
For most of the other Java based software in GCCcore 9.3.0 (and 2020a) we are using Java 11. If this module is likely to be loaded at the same time as other modules, and canu works with Java 11, then it would be good to stick to that Java version.

Reply:
done in f87718d